Gruntfile.js

```diff
@@ -82,12 +95,51 @@ module.exports = function (grunt) {
                     "sudo rm -f -r /var/lib/gpii"
                 ].join("&&")
             },
+            prepareRpmEnv: {
+                command: "dnf install -y nodejs-grunt-cli npm alsa-lib-devel json-glib-devel PackageKit-glib-devel libXrandr-devel libgnome-keyring-devel sudo @development-tools rpmdevtools"
```

Comment:
Ideally the [IDI ansible-nodejs](https://github.com/idi-ops/ansible-nodejs) role should be used so that these dependencies don't have to be tracked in multiple places. This repo uses that role to install these packages like so:

https://github.com/GPII/linux/blob/master/provisioning/vars.yml#L17-L37